examples/basic_bitcoin/test/bitcoin_daemon.ts

```diff
@@ -1,15 +1,9 @@
 import { ChildProcessWithoutNullStreams, spawn } from 'child_process';
 import { existsSync, rmSync } from 'fs-extra';
 
-export async function whileRunningBitcoinDaemon(
-    callback: () => Promise<boolean> | void
-) {
-    const bitcoinDaemon = await startBitcoinDaemon();
-    await callback();
-    bitcoinDaemon.kill();
-}
+export { ChildProcessWithoutNullStreams as BitcoinDaemon };
```

Comment:
Why is this being exported as BitcoinDaemon?

Reply:
to make it nicer to read even though it's slightly deceptive